assets/eip-3525/contracts/extensions/IERC3525Metadata.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+import "../IERC3525.sol";
+import "../openzeppelin/IERC721Metadata.sol";
```

Comment:
```suggestion
import "../IERC3525.sol";
import "IERC721Metadata.sol";
```
Rather than have an external dependency on a specific implementation, it should be left as an exercise for the reader to find an implementation.